docs/introduction/setting-environment-variables.md

```diff
@@ -0,0 +1,74 @@
+# Setting environment variables
+
+In this article describes how to set environments variable in different environment.
+
+The most common uses are: Kubernetes, Docker or native installation.
+
+!!! note
+    
+    Bellow is descibed the easiest way. For better understanding we recomend to find out the documentation of the specific environment.
```

Comment:
I'm afraid this could be mixed up with the application environment (prod/test/dev).

I would refer Docker/Native/K8S as a `platform` or similar word